app/src/main/java/com/paypal/psix/models/Event.java

```diff
@@ -80,6 +80,8 @@ public Event setup() {
         return this;
     }
 
+    public int getNumberOfInvitees() { return this.rsvps().size() - 1; }
```

Comment:
this is used for the description text though. "You have invited X people" is weird if you include yourself in.